cypress/e2e/SubdomainNavigation.cy.ts

```diff
@@ -0,0 +1,221 @@
+import { NFTPageSelector } from '../../src/pages/NFT/selectors';
+import { DomainsPageSelector } from '../../src/pages/Domains/selectors';
+import { SubdomainViewSelector } from '../../src/features/view-subdomains/selectors';
+
+describe('Subdomain Navigation', () => {
+	const rootDomainZna = 'wilder';
+	const subdomainZna = `${rootDomainZna}.snowflake`;
```

Comment:
Noticed a couple times in these tests where an extra variable would come in handy,

```ts
const rootDomainZna = 'wilder';
const childZna = 'snowflake';
const subdomainZna = `${rootDomainZna}.${childZna}`;
```